php/php.editor/nbproject/project.properties

```diff
@@ -78,5 +78,5 @@ test.config.stableBTD.excludes=\
     **/ParamDeclTypes68Test.class,\
     **/SemanticAnalyzerTest.class,\
     **/ToggleBlockCommentActionTest.class
-nashorn.prepend=${basedir}/../libs.nashorn/external/nashorn-02f810c26ff9-patched.jar
-bootclasspath.prepend=${nashorn.prepend}${path.separator}${basedir}/../libs.nashorn/external/asm-all-4.0.jar
+nashorn.prepend=${basedir}/../../webcommon/libs.nashorn/external/com.oracle.js.parser-ba7a8bc42268.jar
```

Comment:
This cross-cluster connection isn't really nice, but if it works... then it is better than nothing.

Reply:
I don't know about this and its workaround... but it seems work.